cmd/govim/testdata/scenario_default/quickfix_empty_files.txt

```diff
@@ -0,0 +1,133 @@
+# Test that we get diagnostics for when there are empty files on disk but
+# in the editor we fill in the content.
+#
+# The scenario involves creating a package p with a simple function DoIt. p is
+# imported by a main package. p also has a test file that exercises DoIt, and
+# an external test file that does the same.
+#
+# Initially all call sites for DoIt incorrectly pass an integer argument,
+# meaning we should have error diagnostics for all call sites. Then we correct
+# the definition of DoIt to take an integer argument at which point all
+# diagnostics should disappear.
+
+# skip 'Pending fix for golang.org/issues/36795'
```

Comment:
I guess this is commented out to provide details to Rebecca & co(?). If so, don't forget to uncomment before merge (or remove if it's fixed).

Reply:
Done